lib/bambora/client.rb

```diff
@@ -20,6 +22,7 @@
 require 'bambora/adapters/response'
 require 'bambora/adapters/json_response'
 require 'bambora/adapters/query_string_response'
+require 'bambora/bank/adapters/payment_profile_response'
```

Comment:
You'll notice that other "builders" and "adapters" don't have nice name spaces like these ones do. I'm going to do a follow up PR that re organizes this stuff a bit.

Reply:
Again, I love that all these patterns are being used here!